File: moban_java/src/main/java/org/jdk/project/dto/config/DeviceRegionBindingUpsertRequest.java
package org.jdk.project.dto.config;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DeviceRegionBindingUpsertRequest {
  private Long deviceId;
  private Long regionId;
  private Long imageId;
  private String coordinate;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/DeviceRegionBindingUpsertRequest.kt
package org.jdk.project.dto.config

data class DeviceRegionBindingUpsertRequest(
    var deviceId: Long? = null,
    var regionId: Long? = null,
    var imageId: Long? = null,
    var coordinate: String? = null,
    var remark: String? = null,
)
